### PR TITLE
add Browser checks

### LINKED ISSUE #16: Make it possible to edit the content of s.coronawarn.app independent of e.coronawarn.app
## Feature description

Currently, the sites https://e.coronawarn.app (for the event-registration) and https://s.coronawarn.app (for RAT) can't be edited independently according to @thomasaugsten in https://github.com/corona-warn-app/cwa-documentation/issues/603#issuecomment-831958316. It should be possible to edit the content of only https://s.coronawarn.app and not of https://e.coronawarn.app and vice versa.

## Problem and motivation
Currently, https://s.coronawarn.app shows exactly the same as https://e.coronawarn.app. 
This could lead to confusing if somebody scanned a QR-Code or clicked on a link from a RAT and then sees "Download the Corona-Warn-App to check in." although he/she only wanted to get his test result for the RAT. (See also https://github.com/corona-warn-app/cwa-documentation/issues/603#issuecomment-831990884)

## Is this something you're interested in working on

After it's possible to edit https://e.coronawarn.app and https://s.coronawarn.app independently of each other, I can provide texts for https://s.coronawarn.app.

### PR DESCRIPTION
**Pull request by thomasaugsten**

### CODE REVIEW

**Comment by Ein-Tim** [comment]
That's also a way to fix #16 (-;

File: src/partials/page-index.html [hunk 1]
```diff
@@ -3,29 +3,33 @@
...
+                    <p class="lead" id="lead">{{page-contents.section-main.content.text}}</p>
+                    <p class="warning d-none" id="warningiOS">{{page-contents.section-main.content.warningiOS}}</p>
+                    <p class="warning d-none"  id="warningAndroid">{{page-contents.section-main.content.warningAndroid}}</p>
+                    <a class="d-none originalUrl" id="originalUrl" href="" target="_blank" rel="noopener noreferrer"> </a>
+    
                     <div class="row">
                         <div class="col col-12 col-md-6 mb-3" id="store_apple">
                             <a href="https://apps.apple.com/de/app/corona-warn-app/id1512595757" target="_blank" rel="noopener noreferrer">
-                                <img class="img-fluid w-100" src="/assets/img/App_Store_Badge_US-UK.svg" alt="Apple App Store Icon" />
+                                <img class="img-fluid w-100" src="/assets/img/App_Store_Badge_DE.svg" alt="Apple App Store Icon" />
```
Comment (svengabr) [inline src/partials/page-index.html:20]: Image missing in repository, resulting in error 404 and no image displayed. @thomasaugsten can you please commit it?

**Review by svengabr — CHANGES_REQUESTED** [review]
Missing image, see my previous comment.

File: src/assets/js/app.js [hunk 2]
```diff
@@ -1,9 +1,28 @@
 document.addEventListener("DOMContentLoaded", function(event) { 
-    if(navigator.userAgent.toLowerCase().indexOf("android") > -1){
+    const originalUrl = sessionStorage.getItem('originalUrl');
+    sessionStorage.removeItem('originalUrl');
+    if (originalUrl) {
+        const originalUrlElement = document.getElementById("originalUrl");
+        originalUrlElement.classList.remove("d-none");
```
Comment (svengabr) [inline src/assets/js/app.js:6]: 2021-05-05 11_34_21-Window

There is a visible printed originalUrl o the website when opening the main website and being redirected to the languageCode `/en` or `/de`

**Review by svengabr — CHANGES_REQUESTED** [review]
How to reproduce

1. open `localhost:8000/`
2. Getting redirected to your language
3. See the url in the center

**Review by svengabr — APPROVED** [review]

**Pull request merged**